js/simpletextbox/Index.svelte

```diff
@@ -0,0 +1,129 @@
+<svelte:options accessors={true} />
+
+<script lang="ts">
+	import type { Gradio } from "@gradio/utils";
+	import { BlockTitle } from "@gradio/atoms";
+	import { Block } from "@gradio/atoms";
+	import { StatusTracker } from "@gradio/statustracker";
+	import type { LoadingStatus } from "@gradio/statustracker";
+	import { tick } from "svelte";
+
+	export let gradio: Gradio<{
+		change: string;
+		submit: never;
+		input: never;
+	}>;
+	export let label = "Textbox";
+	export let elem_id = "";
+	export let elem_classes: string[] = [];
+	export let visible = true;
+	export let value = "";
+	export let placeholder = "";
+	export let show_label: boolean;
+	export let scale: number | null = null;
+	export let min_width: number | undefined = undefined;
+	export let loading_status: LoadingStatus | undefined = undefined;
+	export let value_is_output = false;
+	export let mode: "static" | "interactive";
+	export let rtl = false;
+
+
+	let el: HTMLTextAreaElement | HTMLInputElement;
+	const container = true;
+
+	function handle_change(): void {
+		gradio.dispatch("change", value);
```

Comment:
Why do we dispatch the value for "change" but not for the other events? Can we do:

```
gradio.dispatch("change");
```

Reply:
Yeah. I don't think we use this value at all.